flocker/control/test/configurations/configuration_v1.py

```diff
@@ -0,0 +1,71 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+# Generate a v1 configuration.
+# Commit Hash: 7bd476e2fdc7353018ff1fc446b9b4c76e7c7c17
+
+from json import dumps, JSONEncoder
+from pyrsistent import PRecord, PVector, PMap, PSet
+from uuid import UUID, uuid4
+
+from twisted.python.filepath import FilePath
+
+from flocker.control._model import (
+    Deployment, Dataset, Manifestation,
+    Node, Application, DockerImage, AttachedVolume
+)
+
+_CLASS_MARKER = u"$__class__$"
+
+DATASET = Dataset(dataset_id=unicode(uuid4()),
+                  metadata={u"name": u"myapp"})
+MANIFESTATION = Manifestation(dataset=DATASET, primary=True)
+TEST_DEPLOYMENT = Deployment(
+    nodes=[Node(uuid=uuid4(),
+                applications=[
+                    Application(
+                        name=u'myapp',
+                        image=DockerImage.from_string(u'postgresql:7.6'),
+                        volume=AttachedVolume(
+                            manifestation=MANIFESTATION,
+                            mountpoint=FilePath(b"/xxx/yyy"))
+                    )],
+                manifestations={DATASET.dataset_id: MANIFESTATION})])
+
+
+class _ConfigurationEncoder(JSONEncoder):
```

Comment:
You shouldn't need to copy/paste this (or any) code. Insofar as you gave a commit hash, someone who wants to reproduce the file can checkout that version of the code and run the script with that version of the code.